bitnami/cassandra/4.0/debian-11/rootfs/opt/bitnami/scripts/cassandra/entrypoint.sh

```diff
@@ -18,6 +18,11 @@ set -o pipefail
 
 print_welcome_page
 
+if is_positive_int $CASSANDRA_DELAY_START_TIME && [ $CASSANDRA_DELAY_START_TIME -gt 0 ]; then
+    info "** Delaying Cassandra start by $CASSANDRA_DELAY_START_TIME seconds **"
+    sleep $CASSANDRA_DELAY_START_TIME
+fi
+
```

Comment:
A little suggestion to match with our code styles.

```suggestion
if is_positive_int "$CASSANDRA_DELAY_START_TIME" && [[ "$CASSANDRA_DELAY_START_TIME" -gt 0 ]]; then
    info "** Delaying Cassandra start by ${CASSANDRA_DELAY_START_TIME} seconds **"
    sleep "$CASSANDRA_DELAY_START_TIME"
fi

```